src/frontend/Felles/PersonHeader/PersonHeader.tsx

```diff
@@ -128,7 +57,7 @@ const PersonHeaderComponent: FC<{ data: IPersonopplysninger; behandling?: Behand
 
     const { axiosRequest, erSaksbehandler } = useApp();
     const [fagsakPersonId, settFagsakPersonId] = useState<string>('');
-    const [erMigrert, settErMigrert] = useState(false);
+    const [migrert, settMigrert] = useState(false);
```

Comment:
Hva tenker vi egentlig om booleans ? "erMigrert" vs "migrert" ?

Reply:
støtter bruk av "er", "skal", "har" først i navngivning av booleans for å gjøre det tydeligere at det er true/false

Reply:
Jeg tenker at det er fint å ha med `er` som prefix på variabler hvor navnet er tvetydig slik som for saksbehandler som ovenfor. Ordet `migrert` er i hvertfall for meg ikke så tvetydig så derfor tenker jeg at det er greit uten `er` som prefix også. Men for konsistensens skyld kan jeg godt bytte tilbake!

Reply:
[95ec79b](https://github.com/navikt/familie-ef-sak-frontend/pull/2267/commits/95ec79bd39e168a59e697312e3ec37dac931927d)